packages/desktop/pages/_app.tsx

```diff
@@ -27,13 +28,28 @@ const client = new ApolloClient({
   cache: new InMemoryCache({
     typePolicies: {
       CombatUnitStub: {
-        keyFields: ['id', 'spec'],
+        keyFields: false,
       },
     },
   }),
   link,
 });
 
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
+      staleTime: 600,
```

Comment:
can probably be a bigger number?

Reply:
Potentially but the queries can always override if a higher number is appropriate for their use case -- the log file query overrides this with Inf for example